statler_api/views.py

```diff
@@ -13,13 +15,18 @@ def healthCheck(request):
     return HttpResponse("The API is alive.")
 
 
-def getPlayList(request):
+def getPlayList(request, play_list_id):
     """called when a GET request is sent to /api/play-list/
     returns json of the list of plays
     see https://docs.djangoproject.com/en/1.8/topics/serialization/"""
-    #TODO: get an actual PlayList object instead of all plays
 
-    aPlayDictList = [vars(Play(x)) for x in PlayDAO.objects.all()]
+    if play_list_id == "all":
+        # /api/play_lists/all/ will automatically return all plays
+        aPlayList = PlayDAO.objects.all();
+    else:
+        aPlayList = get_object_or_404(PlayListDAO, url_title=play_list_id).plays.all()
+
+    aPlayDictList = [vars(Play(x)) for x in aPlayList]
```

Comment:
That's a good point about the order. But serializing a PlayList object will result in json that looks like: `{url_title:..., entries: [{index:0, play:{...}}, {index:1, play:{...}}`. That's significantly different from the spec.

Also, the PlayList constructor calls all the other constructors, so we'd be making a copy of our whole object tree every time we get a request. And since the json serializer won't convert objects to json, I've been converting them to dicts first.

Maybe it would be better if every object handled its own serialization. We could stop having DAO and API objects that are essentially the same and add a serializing method to the current DAO objects.